models/contacts.js

```diff
@@ -66,23 +66,23 @@ const removeContact = async (contactId) => {
 const addContact = async (body) => {
   try {
     const { name, email, phone } = body;
-    const contacts = await contactsArray();
+    const contacts = await getContactsArray();
     const findContact = contacts.findIndex(
-      (contact) =>
-        contact.name === name ||
-        contact.email === email ||
-        contact.phone === phone
+      (contact) => contact.email === email || contact.phone === phone
     );
+    const validate = person.validate({
```

Comment:
Validacja lepiej żeby była pierwsza. 
Przychodzi body 
`const response = person.validate(body object)`
`if (response.error) return res.status ... `
Tak mniej więcej napisałem. Ale chodzi żeby szybko sprawdzać i jeśli coś źle to szybko kończyć :)